src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_VALUE = 40;
+
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  if (days >= LONG_TERM) {
+    return RENT_VALUE * days - LONG_TERM_DISCOUNT;
```

Comment:
Good work and clear code! I had the same desigion, but mentor adviced me do not repear (RENT_VALUE * days) 3 times and better make it variable and reuse)